mantidproject/mantid
# FABADA increment

[#11032](http://trac.mantidproject.org/mantid/ticket/11032)

**To test**
- Fit some data with FABADA minimizer.
- Try changing minimizer's properties, check that it doesn't crash.
